GitCommands/RevisionReader.cs

```diff
@@ -194,6 +194,11 @@ public sealed class RevisionReader : IDisposable
                                     !string.IsNullOrWhiteSpace(branchFilter) && !IsSimpleBranchFilter(branchFilter),
                                     "--branches=" + branchFilter
                                 },
+                                {
+                                    refFilterOptions.HasFlag(RefFilterOptions.Branches) &&
+                                    string.IsNullOrWhiteSpace(branchFilter),
```

Comment:
Please keep on the same line
```suggestion
                                    refFilterOptions.HasFlag(RefFilterOptions.Branches) && string.IsNullOrWhiteSpace(branchFilter),
```